style.css

```diff
@@ -23,6 +43,92 @@
     visibility: hidden;
   }
 
+  .blur-mobile {
+    filter: blur(10px);
+  }
+
+  /* Mobile Popup */
+  .mobile-popup {
+    position: fixed;
+    top: 10px;
+    left: 16px;
+    right: 16px;
+    bottom: 10px;
+    height: 98%;
+    z-index: 1;
+  }
+
+  .header-popup {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  .header-popup img {
+    padding-right: 16px;
+  }
+
+  .work-details-popup {
+    display: flex;
+    flex-direction: column;
+    justify-content: start;
+    align-items: flex-start;
+    padding-left: 16px;
+  }
+
+  .work-subtitle-popup {
+    display: flex;
+    gap: 12px;
+    padding-left: 16px;
+    align-items: center;
+  }
+
+  .work-title-popup {
+    font-weight: 400;
```

Comment:
- [ ] Kindly make sure that your design matches that of the template. the description, buttons, and technologies are not visible on the desktop version.
Your design
![Capture](https://user-images.githubusercontent.com/85095702/188755787-e0649a31-2135-4fa6-85d8-a4743b262190.PNG)
Figma template
![figma](https://user-images.githubusercontent.com/85095702/188755825-2cf7e993-779b-451c-9548-4b3eecc7ac9c.PNG)